### PR TITLE
update ci for helpwave tasks

### LINKED ISSUE #145: CI for helpwave tasks
- [ ] add Build and Browserstack CI

### LINKED ISSUE #169: helpwave tasks to Browserstack
- [ ] Update ci to push helpwave task apk and ipa to Browserstack

### PR DESCRIPTION
**Pull request by DasProffi**
closes #169 
closes #145

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -75,20 +83,33 @@ jobs:
...
+          name: ipa-helpwave
           path: ./apps/helpwave/helpwave.ipa
+          
+      - uses: actions/upload-artifact@v3
+        with:
+          name: ipa-tasks
+          path: ./apps/tasks/tasks.ipa    
 
-  browserstack:
+  browserstack-helpwave:
```
Comment (MaxSchaefer) [inline .github/workflows/ci.yaml:112]:
```suggestion
  browserstack:
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -75,20 +83,33 @@ jobs:
 
       - run: melos run intl
       
-      - name: Build ipa
+      - name: package helpwave ipa
```
Comment (MaxSchaefer) [inline .github/workflows/ci.yaml:86]:
```suggestion
      - name: package routing ipa
```

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
Nice! But let's discuss an alternative work-in-progress name for the routing product instead of "helpwave". What do you think about "routing"?

**Comment by DasProffi** [comment]
> Nice! But let's discuss an alternative work-in-progress name for the routing product instead of "helpwave". What do you think about "routing"?

I agree, it's confusing. But this would be something for a team vote and not just the mobile repository.

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -47,15 +47,23 @@ jobs:
...
-      - name: Build apk
-        working-directory: ./apps/helpwave/
+      - run: melos run build-apk
+      
+      - name: rename apks
+        working-directory: ./apps/
         run: |
-          flutter build apk
-
+          mv ./helpwave/build/app/outputs/apk/release/app-release.apk ./helpwave/build/app/outputs/apk/release/app-helpwave.apk
```
Comment (florian-sabonchi) [inline .github/workflows/ci.yaml:55]: I would suggest to call the APK here just tasks  and helpwave

**Review by florian-sabonchi — APPROVED** [review]

**Review by MaxSchaefer — APPROVED** [review]

**Pull request merged**